### PR TITLE
Javascript forms

### PR DESCRIPTION
**Pull request by Cole-Allen**
form_values

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var $sendButton = document.querySelector('#contact-form');
+
+$sendButton.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var obj = {};
+  obj.name = document.forms[0].name.value;
+  obj.email = document.forms[0].email.value;
+  obj.message = document.forms[0].message.value;
+  console.log(obj);
```
Comment (scbowler) [inline javascript-forms/form-values.js:9]: You need to reset the form here

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one update needed. The form needs reset after the form is submitted. Let me know if you have any questions.

File: javascript-forms/form-values.js (hunk 1 above)
Comment (Cole-Allen) [reply javascript-forms/form-values.js:9]: 👍 My bad I thought it was asking to reset in the console

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**